src/ProjectTemplates/BlazorTemplates.Tests/BlazorTemplates.Tests.csproj

```diff
@@ -57,4 +55,12 @@
 
   <!-- Shared testing infrastructure for running E2E template tests -->
   <Import Project="..\TestInfrastructure\PrepareForTest.targets" />
+
+  <Target Name="PublishAssets" AfterTargets="Publish">
+    <ItemGroup>
+      <_browsers Include="$(USERPROFILE)\AppData\Local\ms-playwright\**\*.*" />
```

Comment:
Need to figure out how to publish/install correct binaries for linux/macos

Reply:
There are two things you can do:
* With standalone CLI that gets published with the publish output you could run `install` and that would provision the browsers at that point (for the specific platform you are targeting)
* There's a dotnet tool `playwright-sharp-tool` that can be used to install the browsers

Reply:
Looks like we might be able to just install the browsers as part of the helix bootstrapper via `await Playwright.InstallAsync();` that seems best since then we don't have to figure out how to select the right platform since we only publish off of windows